torch/csrc/utils/disable_torch_function.cpp

```diff
@@ -125,3 +126,102 @@ PyObject* THPModule_disable_torch_function(PyObject *self, PyObject *a) {
   return result;
   END_HANDLE_TH_ERRORS
 }
+
+// Makes sure that we don't check for __torch_function__ on basic Python types
+static bool is_basic_python_type(PyTypeObject *tp)
+{
+  return (
```

Comment:
A tangent, but this is a place I see `constexpr unordered_map` coming in handy in the future.

Reply:
I actually tried `ska::flat_hash_set` but it was slower than just doing the checks directly.